ow_faults_injection/include/ow_faults_injection/FaultInjector.h

```diff
@@ -80,7 +83,10 @@ class FaultInjector
 
   // Camera
   ros::Subscriber m_camera_raw_sub;
-  ros::Publisher m_camera_trigger_remapped_pub;
+  ros::Subscriber m_camera_info_sub;
```

Comment:
Unrelated to this change, but a minor suggestion: since all that is ever done with subscribers is a single subscription, there is no need for individual data members for each.  You can replace all of them with a list of `ros::Subscriber` that you just push into with each subscription.  The subscribers do need to exist, but they can be anonymous.